proteus/mprans/cMCorr.pyx

```diff
@@ -6,498 +7,479 @@ from proteus.Transport import OneLevelTransport
-                                       int offset_u, int stride_u,
-                                       double* globalResidual,
-                                   int nExteriorElementBoundaries_global,
-                                   int* exteriorElementBoundariesArray,
-                                   int* elementBoundaryElementsArray,
-                                   int* elementBoundaryLocalElementBoundariesArray,
-                                   double* H_dof)
+                               double* mesh_grad_trial_ref,
+                               double* mesh_dof,
+                               int* mesh_l2g,
+                               double* dV_ref,
+                               double* u_trial_ref,
+                               double* u_grad_trial_ref,
+                               double* u_test_ref,
+                               double* u_grad_test_ref,
+                               double* mesh_trial_trace_ref,
+                               double* mesh_grad_trial_trace_ref,
+                               double* dS_ref,
+                               double* u_trial_trace_ref,
+                               double* u_grad_trial_trace_ref,
+                               double* u_test_trace_ref,
+                               double* u_grad_test_trace_ref,
+                               double* normal_ref,
+                               double* boundaryJac_ref,
+                               int nElements_global,
+                               double useMetrics,
+                               double epsFactHeaviside,
+                               double epsFactDirac,
+                               double epsFactDiffusion,
+                               int* phi_l2g,
+                               double* elementDiameter,
+                               double* nodeDiamaetersArray,
+                               double* phi_dof,
+                               double* q_phi,
+                               double* q_normal_phi,
+                               double* ebqe_phi,
+                               double* ebqe_normal_phi,
+                               double* q_H,
+                               double* q_u,
+                               double* q_n,
+                               double* ebqe_u,
+                               double* ebqe_n,
+                               double* q_r,
+                               double* q_porosity,
+                               int offset_u, int stride_u,
+                               double* globalResidual,
+                               int nExteriorElementBoundaries_global,
+                               int* exteriorElementBoundariesArray,
+                               int* elementBoundaryElementsArray,
+                               int* elementBoundaryLocalElementBoundariesArray,
+                               double* H_dof)
+        void FCTStep(int NNZ,
```

Comment:
I moved these to the bottom as we discussed, but I thought you hadn't modified MCorr* with FCT/EV. Did you mean to cut these out once you created MCorr2?

Reply:
I think that MCorr2.h shouldn't be there. In github it says that the file was created 2 days ago. This makes me think that I added it in the first cleanup of mprans from one of my local branches. In other words, in some local branch I was experimenting with something and created the MCorr2.h file and two days ago I accidentally added it while I was doing a clean up of all .h files in mprans. So we can delete MCorr2.h

Reply:
OK, so your  FCT and mass matrix stuff that you've added to MCorr is supposed to be in there, correct?

Reply:
Correct. My changes to the mass correction model are in MCorr.h (and the respective pyx and py files) so we must delete MCorr2.h and cMCorr2.pyx.